src/cljs/org/broadinstitute/firecloud_ui/common/gcs_file_preview.cljs

```diff
@@ -26,6 +26,13 @@
          [:div {:style {:width 700 :overflow "auto"}}
           (labeled "Google Bucket" (:bucket-name props))
           (labeled "Object" (:object props))
+          [:div {:style {:marginTop "1em"}}
+           (if (> data-size 1800) (str "Last " (:previewLineCount @state) " lines of log are shown. Use the link below to view the full log.") "Log:")
+           (react/create-element
+             [:div {:ref "preview" :style {:marginTop "1em" :whiteSpace "pre-wrap" :fontFamily "monospace"
+                                           :fontSize "90%" :overflow "auto" :maxHeight 206
```

Comment:
Nit: only need `:overflowY` rather than `:overflow`. `:maxHeight 206` begs for a comment.